app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -101,6 +100,8 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
     TextView title;
     @BindView(R.id.media_detail_caption)
     TextView mediaCaption;
+    @BindView(R.id.media_detail_depiction)
+    TextView mediaDepiction;
```

Comment:
there can be several depictions, please implement in a similar way to categories (which are multiple too)